lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -699,30 +690,27 @@ private boolean shouldDrawMultiline() {
   }
 
   private void drawMultinlineTransition(
-      @NonNull Canvas canvas, float currentExpandedX, float x, float y, float ascent) {
+      @NonNull Canvas canvas, float currentExpandedX, float y) {
     int originalAlpha = textPaint.getAlpha();
     // positon expanded text appropriately
     canvas.translate(currentExpandedX, y);
     // Expanded text
     textPaint.setAlpha((int) (expandedTextBlend * originalAlpha));
     textLayout.draw(canvas);
 
-    // position the overlays
-    canvas.translate(x - currentExpandedX, 0);
-
     // Collapsed text
     textPaint.setAlpha((int) (collapsedTextBlend * originalAlpha));
-    canvas.drawText(
-        textToDrawCollapsed, 0, textToDrawCollapsed.length(), 0, -ascent, textPaint);
+    canvas.drawText(textToDrawCollapsed, 0, textToDrawCollapsed.length(), 0,
+        textLayout.getLineBaseline(0), textPaint);
```

Comment:
Since `canvas.drawText(...)` expects the y to be the text baseline and canvas here is translated to top of the text, we get the baseline of the first line. 

Now that I look at it, it is probably true, that `-ascent == textLayout.getLineBaseline(0)`, but the latter is IMO clearer.